models/organization/org_user.go

```diff
@@ -77,6 +77,27 @@ func IsOrganizationMember(ctx context.Context, orgID, uid int64) (bool, error) {
 		Exist()
 }
 
+// IsOrganizationsMember returns a map with key of orgID and value is true if given user is member of organization.
+func IsOrganizationsMember(ctx context.Context, orgIDs []int64, uid int64) (map[int64]bool, error) {
```

Comment:
The permission of organization is complex. Not all users can see all repos of the org. 
e.g. a member in a team which has no repo access, then members can also see the actions of some private repos they can not access.
![image](https://github.com/go-gitea/gitea/assets/18380374/bea3617f-9690-455e-a74f-1e4c5eed541f)

Not sure whether we need to consider this case, but actually it exists.
And maybe we can add some integration tests for checking all cases of different permissions.

Reply:
Any updates?